packages/server/customer-os-api/graph/schemas/dashboard.graphqls

```diff
@@ -1,6 +1,6 @@
 extend type Query {
     """
-    sort.By available options: CONTACT, EMAIL, ORGANIZATION, LOCATION
+    sort.By available options: CONTACT, EMAIL, ORGANIZATION, LOCATION, RELATIONSHIP, STAGE
```

Comment:
if different values are allowed for different methods,

shouldn't they be made into separate enumerations instead of just comments?